src/lib/core/components/visualizations/implementations/BoxplotVisualization.tsx

```diff
@@ -465,6 +477,7 @@ function BoxplotViz(props: VisualizationProps) {
       // show/hide independent/dependent axis tick label
       showIndependentAxisTickLabel={true}
       showDependentAxisTickLabel={true}
+      dependentAxisRange={dependentAxisRange}
```

Comment:
Here is the fix for the lost prop!

Reply:
Hi @moontrip  - the prop is definitely here.  Must have been some cache issue on your side?